zerver/management/commands/generate_invite_links.py

```diff
@@ -29,9 +29,19 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         # type: (*Any, **Any) -> None
         duplicates = False
+
+        realm = None
+        string_id = options["string_id"]
+        if string_id:
```

Comment:
We should probably add a check whether string_id is empty is or not. Else we will get something like this if user forgot to  pass the realm. 

`The realm None doesn't exist yet, please create it first.
`